### DISCUSSION TITLE
Donwload older version APK for Andriod

### DISCUSSION THREAD

**Discussion by krummelhoeve** [rustdesk/rustdesk#8223] [Q&A] [answered]
I'd like to download 1.1.9 or 1.1.8 as APK just like newer versions (1.2.x), but I can't find any apk download link in https://github.com/rustdesk/rustdesk/releases. Is there someone that could help me out?

I upgraded to 1.2.3 on an older Samsung tablet, but now Rustdesk constantly crashes after 1 minute or so. It did work better with an older version.

**Comment by xlionjuan** [comment, marked as answer]
https://www.apkmirror.com/apk/carriez-studio/

**Reply by krummelhoeve** [reply]
Thank you very much xlionjuan!
